### DISCUSSION TITLE
Filament on production & Local (Problems with creating/editing users)

### DISCUSSION THREAD

**Discussion by DevMushref** [filamentphp/filament#5439] [Help] [answered]
I am facing a few issues with the User Resource and I need assistance solving them:

1- When I create a user through `php artisan` and login, it works. However, when I change the user's password through the admin dashboard and try to login I get a credentials error that it doesn't exist (It does) and the password in the database is not encrypted after being edited.

2- I am a `super_admin`, why does filament ask me to fill the password input of the user I'm trying to edit? Logically I wouldn't know it. It does save the changes if I submit a random text in the password field.

3- When I create a user within the filament dashboard and try to login, I get the credentials errors as well.

**Note: I haven't changed anything in the user model & resource.**

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/tricks/password-form-fields
